main.py

```diff
@@ -1,9 +1,11 @@
-from datetime import datetime
+import time
 
+work_time_duration = 5
```

Comment:
We're going to want to make this either configurable or longer than five seconds at a later date!

Reply:
That's fair.